WordPress/Classes/ViewRelated/Notifications/Comment Moderation/CommentModerationOptionsView.swift

```diff
@@ -0,0 +1,111 @@
+import SwiftUI
+import DesignSystem
+
+struct CommentModerationOptionsView: View {
+    private struct Item {
+        let title: String
+        let iconName: IconName
+        let tintColor: Color
+    }
+
+    private let items = [
+        Item(
+            title: Strings.approveTitle,
+            iconName: .checkmarkCircle,
+            tintColor: .DS.Foreground.brand(isJetpack: true)
+        ),
+        Item(
+            title: Strings.pendingTitle,
+            iconName: .clock,
+            tintColor: .DS.Foreground.secondary
+        ),
+        Item(
+            title: Strings.trashTitle,
+            iconName: .trash,
+            tintColor: .DS.Foreground.warning
+        ),
+        Item(
+            title: Strings.spamTitle,
+            iconName: .exclamationCircle,
+            tintColor: .DS.Foreground.error
+        ),
+    ]
+
+    var body: some View {
+        VStack(alignment: .leading, spacing: .DS.Padding.medium) {
+            title
+            optionsVStack
+        }
+        .padding(.horizontal, .DS.Padding.double)
+        .background(Color.DS.Background.primary)
+    }
+
+    private var title: some View {
+        Text(Strings.title)
+            .font(.DS.Body.Emphasized.large)
+            .foregroundStyle(Color.DS.Foreground.primary)
+    }
+
+    private var optionsVStack: some View {
+        VStack(spacing: .DS.Padding.medium) {
+            ForEach(items, id: \.title) { item in
+                optionHStack(item: item)
+            }
+        }
+    }
```

Comment:
@alpavanoglu I haven't tested the PR yet, but I don't see a button here to select an option?

Reply:
Resolved in https://github.com/wordpress-mobile/WordPress-iOS/pull/23090/commits/9ccbdcee0e9fc1630a63a2a1ed9dcd16da4af63f

Wrapped the HStack in a Button